README.md

```diff
@@ -11,41 +11,41 @@ The `RSCT` Operator enables RSCT on a cluster.
 ### Installing the `RSCT` Operator
 Run the following command to deploy the `RSCT` Operator:
 ```sh
-make deploy
+oc create -k "https://github.com/ocp-power-automation/rsct-operator/config/default/?ref=main"
 ```
 
 ### Create `RSCT` resource
-1. Switch to the `rsct-operator-system` namespace
+1. Create the cluster role binding and `RSCT` resources
    ```sh
-   oc project rsct-operator-system
-   ```
-
-2. Create the cluster role binding and `RSCT` resources
-   ```sh
-   oc apply -f config/samples/rsct_v1alpha1_rsct.yaml
+   oc create -k "https://github.com/ocp-power-automation/rsct-operator/config/samples/?ref=main"
```

Comment:
seems to be duplicate instructions if you see above, needs refactoring..

Reply:
These are different files to deploy operator and then custom resource.